AUTHORS

```diff
@@ -28,3 +28,4 @@ Contributors
 * Soukaina Hikma
 * Mudit Loya
 * Priyabrata Mondal
+* Farouk ECH-CHAREF
```

Comment:
We do not capitalize last names.

Reply:
Sorry, I meant that we do not use ALL CAPS for names.

Reply:
Would this work for you?
```suggestion
* Farouk Ech-Charef
```

Reply:
Yes it's fine, thanks for the clarification

Reply:
Should I commit the suggested change, or is it for you to decide?

Reply:
You can commit the suggested change